samples/sampler/stories/default/QuickGuidePanels.js

```diff
@@ -13,31 +20,72 @@ export default {
 	component: 'QuickGuidePanels'
 };
 
-export const _QuickGuidePanels = (args) => (
-	<QuickGuidePanels
-		current={args['current']}
-		nextButtonVisibility={args['nextButtonVisibility']}
-		onBack={action('onBack')}
-		onChange={action('onChange')}
-		onClose={action('onClose')}
-		onNextClick={action('onNextClick')}
-		onPrevClick={action('onPrevClick')}
-		prevButtonVisibility={args['prevButtonVisibility']}
-		total={args['total']}
-	>
-		<QuickGuidePanels.Panel>
-			This is Panel 0
-		</QuickGuidePanels.Panel>
-		<QuickGuidePanels.Panel>
-			<div style={{height: '100%', width: '100%', backgroundColor: 'grey'}}>
-				This is Panel 1
-			</div>
-		</QuickGuidePanels.Panel>
-		<QuickGuidePanels.Panel aria-label={'aria test'}>
-			This is Panel 2
-		</QuickGuidePanels.Panel>
-	</QuickGuidePanels>
-);
+export const _QuickGuidePanels = (args) => {
+	const screenWidth = window.screen.width;
+	const screenHeight = window.screen.height;
```

Comment:
How about adding `?` for undefined checking?

Reply:
It's fixed